ivy_tests/test_ivy/test_frontends/test_numpy/test_creation_routines/test_numerical_ranges.py

```diff
@@ -193,6 +217,67 @@ def test_numpy_meshgrid(
     )
 
 
-# TODO
 # mgrid
+@handle_cmd_line_args
+@given(range=_get_range_for_grid())
+def test_numpy_mgrid(
+    range,
+    fw,
+):
+    ivy.set_backend(fw)
```

Comment:
this line should not be needed. It should be handled by `@handle_cmd_line_args`, are you sure it's required?

Reply:
oh right, I just checked and it's done with `f.use`, just removed them! thanks!